plugins/chat/app/models/category_channel.rb

```diff
@@ -17,7 +17,11 @@ def allowed_group_ids
     category.secure_group_ids.to_a.concat(staff_groups)
   end
 
-  def title(_)
+  def title(_ = nil)
     name.presence || category.name
   end
+
+  def slug
+    title.truncate(100).parameterize
```

Comment:
Note this is a temporary solution while we work on https://github.com/discourse/discourse/pull/18879, this is why there are no tests about it, it shouldn't live more than a week